### ISSUE TITLE
Support multi cycles

### ISSUE THREAD

**Issue by Amzani** [shapeitapp/shapeitapp#83]
Even though ShapeUp methodology recommends to ignore a bet after the end of a cycle, but OSS is unpredictable and it's difficult to commit to complete the scope by the end of the cycle.

## Solution

Provide ability to select multiple cycles so we can have history

**Comment by Amzani** [comment]
@fmvilas you probably have a better idea ?

**Comment by fmvilas** [comment]
Good point. I don't know to be honest. This means you should be able to select multiple cycles then. Also, you may not use two consecutive cycles. And there should be a limitation that, for instance, after cycle 3 you can't assign it to cycle 1. You can only assign it to another one but a future/current one (never a past one). Might be as easy as making the Cycle field a comma-separated list 🤔
